examples/tutorials/tutorial_v2_4_0_all_sets_E3SM_machines.py

```diff
@@ -0,0 +1,221 @@
+import os
+
+from acme_diags.parameter.area_mean_time_series_parameter import (
+    AreaMeanTimeSeriesParameter,
+)
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.diurnal_cycle_parameter import DiurnalCycleParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.parameter.qbo_parameter import QboParameter
+from acme_diags.parameter.streamflow_parameter import StreamflowParameter
+from acme_diags.parameter.arm_diags_parameter import ARMDiagsParameter
+from acme_diags.run import runner
+
+
+def run_compy(html_prefix):
+    # Run the following first:
+    # srun --pty --nodes=1 --time=01:00:00 /bin/bash
+    # source /share/apps/E3SM/conda_envs/load_latest_e3sm_unified.sh
+    ref_data_prefix = "/compyfs/e3sm_diags_data/obs_for_e3sm_diags"
+    test_data_prefix = "/compyfs/e3sm_diags_data/test_model_data_for_acme_diags"
+
+    test_data_prefix2 = "/compyfs/fors729"
+
+    d = dict()
+
+    d["obs_climo"] = os.path.join(ref_data_prefix, "climatology/")
+    d["test_climo"] = os.path.join(test_data_prefix, "climatology/")
+
+    d["obs_ts"] = os.path.join(ref_data_prefix, "time-series/")
+    d["test_ts"] = os.path.join(test_data_prefix, "time-series/E3SM_v1/")
+
+    d["dc_obs_climo"] = d["obs_climo"]
+    d["dc_test_climo"] = os.path.join(
+        test_data_prefix, "climatology/diurnal_cycle_climatology/"
+    )
+
+    d["streamflow_obs_ts"] = d["obs_ts"]
+    d["streamflow_test_ts"] = os.path.join(
+        test_data_prefix2, "time-series/streamflow_ts_postprocessed/"
+    )
+
+    d["arm_obs"] = os.path.join(ref_data_prefix, "arm-diags-data/")
+    d["arm_test"] = os.path.join(test_data_prefix2, "20210122.F2010.armsites/")
```

Comment:
I don't have write permission for `/compyfs/e3sm_diags_data/test_model_data_for_acme_diags` so I put the arm-diags test data in `/compyfs/fors729/20210122.F2010.armsites`